hail/Makefile

```diff
@@ -532,6 +531,7 @@ clean: clean-env clean-libs native-lib-clean mill-clean
 	rm -rf $(SCALA_BUILD_INFO)
 	rm -rf $(PYTHON_VERSION_INFO)
 	rm -rf python/hail/docs/change_log.rst
+	rm -rf src/main/resources/build-info.properties
```

Comment:
Can we leave a comment here explaining that this is deleting something that's only created by the old gradle build?